src/mapcat-mapview-init.js

```diff
@@ -58,7 +73,7 @@ function initView (type, layers, userid, callback, lang, schema, scale) {
         headers: {
             'Content-Type': 'application/json',
             'Content-Length': Buffer.byteLength(postData),
-            'X-Api-Key': userid
+            'X-Api-Key': accesToken
```

Comment:
az accessToken-re is csinálnék tesztet, hogy nem undefined, nem null és string típusú